.github/workflows/update-compat-data.yml

```
@@ -0,0 +1,76 @@
+name: Update Test262 parser tests
+env:
+  YARN_ENABLE_SCRIPTS: false # disable post-install scripts
+on:
+  workflow_dispatch:
+    inputs: {}
+  schedule:
+    - cron: "0 0 * * 5"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: tc39/test262
+          path: build/test262
```

```suggestion
          repository: kangax/compat-table
          path: packages/babel-compat-data/build/compat-table
```

Since `compat-table` is hosted on GitHub, we can check out the repo to the specific folder using `actions/checkout` and get the `HEAD` commit sha1 from the cloned git repository, like we did for `test262`

https://github.com/babel/babel/blob/165b7352b71169d636731da9cd4c16886939b39a/scripts/parser-tests/get-test262-version.sh#L7-L9

By doing so we save extra http request to GitHub API and one more dev deps.